app/models/survey.rb

```diff
@@ -7,6 +7,7 @@ class Survey < ApplicationRecord
   has_many :questions, through: :visits
```

Comment:
This is great, as is the commit message (explains everything 👍) but the commit msg should wrap at [~72 characters](https://gds-way.cloudapps.digital/standards/source-code.html#structure)